.github/start-wasm-example/tests/wasm_example.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('http://localhost:8000/');
+});
+
+test.describe('WASM example', () => {
+  test('Wait for success', async ({ page }, test_info) => {
+    let start = new Date().getTime();
+
+    let found = false;
+    while (new Date().getTime() - start < 300_000) {
```

Comment:
Can we pull this out these magic numbers into constants so future editors know which values can be safely tweaked?